core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -175,6 +175,10 @@ private[spark] class DAGScheduler(
   // TODO: Garbage collect information about failure epochs when we know there are no more
   //       stray messages to detect.
   private val failedEpoch = new HashMap[String, Long]
+  // There will be a regression when an executor lost and then causes 'fetch failed'.
```

Comment:
`There will be a regression` is misleading because this mean this PR causes a regression.